ssn/integrated/informativeExamples/InkBird-IBS-TH2-instance.ttl

```diff
@@ -0,0 +1,256 @@
+    prov:atLocation <acmeBreweryCo> ;
+    prov:generated <00001/acmePorterSixPack> .
+
+# These definitions look redundant; but they represent the specific, physical instantiation of the sensors.
+
+
+# Batteries are a consumable component of the platform and must be replaced from time to time. Since the
+# battery health sensor is observing a new battery pack each time it is replaced, we create a new digital
+# twin to be related to the observation. This means that device power consumption and the performance of
+# different battery brands can also be analyzed.
+
+<2024/02/batteryPack> a equipment:IBSTH2Battery;
+    rdfs:label "IBSTH2 Battery. 2 AAA Batteries. 3.0V Nominal."@en ;
+    skos:editorialNote "Alice doesn't quite believe the battery lifetime claims, so she's tracking every set of batteries she uses and a new battery instances is generated each time." ;
+    gs1:brand <AcmeBatteryBrand> ;
+    dcterms:created "2024-02-20T01:35:45Z"^^xsd:dateTime .
+
+<AcmeBatteryBrand> a gs1:Brand ;
+    rdfs:label "Acme Branded Battery"@en ;
+    gs1:brandName "Acme"@en .
+
+<12345/BatterySensor> a equipment:IBSTH2BatteryVoltageSensor;
+    rdf:comment "This is the instance of the physical sensor instance."@en ;
+    sosa:observes <12345/BatteryHealth> .
+
+<12345/BatteryHealth> a equipment:IBSTH2BatteryHealth ;
+    ssn:isPropertyOf <2024/02/batteryPack> .
+
+<12345/HumiditySensor> a equipment:IBSTH2HumiditySensor ;
+    rdf:comment "This is the instance of the humidity sensor instance."@en .
+
+<12345/TemperatureSensor> a equipment:IBSTH2TemperatureSensor ;
+    rdf:comment "This is the instance of the temperature sensor instance."@en .
+
+<TemperatureOfProduct> rdfs:subClassOf equipment:airTemperature ;
+    ssn:isPropertyOf <00001/acmePorterSixPack> .
+
+<airRelativeHumidityInProductPackaging> rdfs:subClassOf equipment:airRelativeHumidity;
+    ssn:isPropertyOf <00001/acmePorterSixPack> .
+
+<airRelativeHumidityInProductPackaging>
+    schema:unitCode unit:PERCENT ;
+    a sosa:ObservableProperty, ssn:Property ;
+    rdfs:label "Humidité relative de l'emballage du produit"@fr, "Relative Humidity of Product Packaging"@en, "Relative Luftfeuchtigkeit der Produktverpackung"@de ;
+    skos:definition "The amount of water vapor present in air around the product packaging"@en .
+
+<BeerTemperature>
+    schema:unitCode unit:DEG_C ;
+    a sosa:ObservableProperty, ssn:Property ;
+    rdfs:label "Biertemperatur"@de, "Beer Temperature"@en, "Température de la bière"@fr ;
+    skos:definition "Temperature of the Beer."@en .
+
+#
+# Sensor activates and records temperature while in the brewery cooler.
+    
+<observation1a> a sosa:Observation ;
+    rel:own <acmeBreweryCo> ;
+    sosa:observedProperty equipment:airTemperature ;
+    sosa:hasUltimateFeatureOfInterest <BeerTemperature> ; 
+    sosa:madeBySensor <12345/TemperatureSensor>  ;
+    sosa:hasFeatureOfInterest <00001/acmePorterSixPackAirSample> ;
```

Comment:
[ldesousa](https://github.com/ldesousa) Can you please review? This example becomes annoying since the only reference to the beer carton is through the air sample instance.